common/src/main/java/com/microsoft/identity/common/internal/cache/SharedPreferencesFileManagerSimpleCacheImpl.java

```diff
@@ -142,6 +173,15 @@ public String getName() {
             @Override
             public Boolean call() {
                 final Set<T> allMetadata = new HashSet<>(getAll());
```

Comment:
Hm.  Why not translate it into a map, and just separate keys from values?  I mean, since a set is a map…. That seems less strange to me than excluding something from equality and then doing work to change the set contract…

Reply:
Not sure I follow... this may not be clear, but there are no keys for this data. It's just a collection of values on disk

Reply:
Sure, but we're free to interpret that collection any way we'd like.  If we want to read it in order to create a Map instead of a Set, that's on us.  I was just thinking that instead of making the Set act like a Map, just arrange the data so it's in a Map instead.